lib/check.h

```diff
@@ -26,6 +26,7 @@
 #include "tokenize.h"
 #include "settings.h"
 #include "errorlogger.h"
+#include "cwe.h"
```

Comment:
You can squash commits to a single one and do a force push. The pull-request gets updated automatically then.